docker-compose.yml

```diff
@@ -684,7 +684,7 @@ services:
     #   CONAN: gcc11, gcc11-armv7, ...
     #   See https://github.com/conan-io/conan-docker-tools#readme for
     #   available images.
-    image: conanio/${CONAN}
+    image: conanio/${CONAN}:1.62.0
```

Comment:
As this seems to work, once the build is successful we probably want to make it slightly more configurable with an environment var, like:
```diff
diff --git a/.env b/.env
index 6746892..0e94de5 100644
--- a/.env
+++ b/.env
@@ -100,7 +100,8 @@ VCPKG="501db0f17ef6df184fcdbfbe0f87cde2313b6ab1"    # 2023.04.15 Release
 # use pulled built images in dev/tasks/python-wheels/github.windows.yml.
 PYTHON_WHEEL_WINDOWS_IMAGE_REVISION=2023-08-02
 
-# Use conanio/${CONAN} for "docker-compose run --rm conan". See
-# https://github.com/conan-io/conan-docker-tools#readme for available
-# images.
-CONAN=gcc10
+# Use conanio/${CONAN_BASE}:{CONAN_VERSION} for "docker-compose run --rm conan".
+# See https://github.com/conan-io/conan-docker-tools#readme and
+# https://hub.docker.com/u/conanio for available images.
+CONAN_BASE=gcc10
+CONAN_VERSION=1.62.0
diff --git a/docker-compose.yml b/docker-compose.yml
index a08345c..45c4e85 100644
--- a/docker-compose.yml
+++ b/docker-compose.yml
@@ -681,10 +681,11 @@ services:
     # Usage:
     #   docker-compose run --rm conan
     # Parameters:
-    #   CONAN: gcc11, gcc11-armv7, ...
+    #   CONAN_BASE: gcc11, gcc11-armv7, ...
+    #   CONAN_VERSION: 1.62.0
     #   See https://github.com/conan-io/conan-docker-tools#readme for
     #   available images.
-    image: conanio/${CONAN}
+    image: conanio/${CONAN}:${CONAN_VERSION}
     user: root:root
     shm_size: *shm-size
     ulimits: *ulimits
```
I should have proposed this at the very beginning :) sorry

Reply:
Yes it works :) 

It's time for me to sleep now. I will update it tomorrow.